node/malus/src/variants/suggest_garbage_candidate.rs

```diff
@@ -255,14 +273,15 @@ impl OverseerGen for BackGarbageCandidateWrapper {
 		RuntimeClient::Api: ParachainHost<Block> + BabeApi<Block> + AuthorityDiscoveryApi<Block>,
 		Spawner: 'static + SpawnNamed + Clone + Unpin,
 	{
-		let inner = Inner { map: std::collections::HashMap::new() };
-		let inner_mut = Arc::new(Mutex::new(inner));
-		let note_candidate =
-			NoteCandidate { inner: inner_mut.clone(), spawner: SpawnGlue(args.spawner.clone()) };
-
+		let note_candidate = NoteCandidate {
+			spawner: SpawnGlue(args.spawner.clone()),
+			percentage: f64::from(self.opts.percentage),
+		};
+		let validation_probability = 100.0;
```

Comment:
This argument is passed to `ReplaceValidationResult::new()`, and used to initialize the `Bernoulli` of the `ReplaceValidationResult` struct. I think a better term here actually is `back_invalid_probability` since the idea is to make sure that the probability of backing malicious candidates (as defined in `common.rs`) is always set to 100% when starting the `suggest-garbage-candidate` nemesis, whilst the probability the same node would suggest a garbage candidate is determined via the `--percentage` option. Is this a correct assumption to make? My reason for thinking this is because it would make little sense to let the suggestor not back its own, or another malicious suggestor's garbage suggestion in case it receives one. However, I am not 100% of how someone suggesting a garbage candidate would participate in backing its own, or another malicious node's garbage suggestion.

F.ex, consider a scenario with 2 Malus nodes in a test net:
```
t0, Malicious A: suggests a garbage candidate with some probability `p`
t1, Malicious B: skips the suggestion of a garbage candidate with some probability `1-p`
t2, Malicious B: receives A's garbage suggestion, and backs it with probability of 100% (default as assumed here)
t3, Malicious A: receives its own suggestion and backs it with probability of 100% (default as assumed here)
```
In principle, if we can adjust how many malicious suggestions we produce with the `--percentage` parameter, then it seems we could benefit from using the same node to back malicious candidates with some additional probability.

Reply:
This sounds goot in theory, but in practice it would make the UX clunky - requires deeper knowledge of how things work under the hood and enables higher misconfiguration potential.

Reply:
Well, we could separate between `--backing-percentage` and `--suggest-percentage`. Usage would be:

```
$ malus suggest-garbage-candidate --backing-percentage 50 --suggest-percentage-100
```

for example.